api/utility-functions/api-utils.js

```diff
@@ -1,7 +1,7 @@
 const format = require("pg-format");
 const connection = require("../../db/connection");
 
-exports.checkArticleExists = (table_name, column_name, value) => {
+exports.checkExists = (table_name, column_name, value) => {
```

Comment:
I can't see from this commit if that function has been tested but just make sure it has been

Reply:
You could think about using a model you already have to check if the article exists. Also, a comment cannot be inserted without a username that exists so you should get a psql error in that case.